### ISSUE TITLE
Images are not output to dist

### ISSUE THREAD

**Issue by cezaraugusto** [rothsandro/eleventy-notes#47]
Hello there, great project! 

I'm using eleventy-notes to build my documentation website (https://docs.extensioncreate.com) but image paths are not output to `dist/`. Here's a screenshot of the image missing ([GitHub can find it tho](https://github.com/cezaraugusto/docs.extensioncreate.com/blob/main/index.md)).

<img width="501" alt="Screenshot 2024-03-25 at 14 12 34" src="https://github.com/rothsandro/eleventy-notes/assets/4672033/0fe018c3-5d55-409a-b188-77f83ac77400">

Maybe there a rule to where I can add files to? I created an `assets/` folder to host them, which works fine but only during development.

<img width="281" alt="Screenshot 2024-03-25 at 14 15 33" src="https://github.com/rothsandro/eleventy-notes/assets/4672033/8e5f1bb4-0445-4509-ac43-99b512c47b04">

How can I make it work?

**Comment by rothsandro** [comment]
Thanks for reporting this. Looks like images were completely broken in production, really weird. I fixed it (hopefully) and released v0.23.1. Please update to the latest version and let me know if you still encounter problems.

Note: In some files the image path is in quotation marks which is wrong (and also doesn't work on GitHub), make sure you remove them:

```diff
- ![WASM Sample Extension]("./assets/wasm.png")
+ ![WASM Sample Extension](./assets/wasm.png)
```

**Comment by cezaraugusto** [comment]
Working great after the update, thanks!
